components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/Editor.java

```diff
@@ -288,6 +288,17 @@ public void setChannelAcquisitionData(int index,
 	 */
 	public void download(File file, boolean override);
 
+    /**
+     * Downloads the archived files, preserving the original folder structure
+     * 
+     * @param file
```

Comment:
Should be path, okay for rebase but try not to introduce more Javadoc warnings :smiley: Can enable Javadoc "yellow warnings" in Eclipse.